swagger.yaml

```diff
@@ -3117,5 +3379,36 @@ components:
           type: string
           description: Last two characters of EWC (European Waste Code) code
           example: 01
+        metadata:
+          $ref: "#/components/schemas/Metadata"
+
+    Hazard:
```

Comment:
This is a bit generic term, could it be renamed to HazardousSurveyMaterial or HazardousWaste or something more specific?